src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesSynchronisationService.kt

```diff
@@ -58,6 +73,11 @@ class CaseNotesSynchronisationService(
 //     "caseNoteText").isEqualTo("A note")
 //     "amended").isEqualTo("false")
 //     "occurrenceDateTime") assertThat(LocalDateTime.parse(it)).isCloseTo(now, within(2, ChronoUnit.MINUTES)) }
+
+    telemetryClient.trackEvent(
+      "casenotes-synchronisation-updated-success",
+      event.toTelemetryProperties(mapping.dpsCaseNoteId),
```

Comment:
Isn't this already stored in variable `telemetry`?

Reply:
yep its confusing, ok ill tidy it up